bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -2188,6 +2189,29 @@ public long getAuditorPeriodicBookieCheckInterval() {
         return getLong(AUDITOR_PERIODIC_BOOKIE_CHECK_INTERVAL, 86400);
     }
 
+    /**
+     * Set the regularity at which the auditor will run a metadata check of all
+     * write closed/fenced ledgers. This should not be run very often, and at
+     * most, once a day. Setting this to 0 will completely disable the periodic
+     * metadata check.
```

Comment:
yes this metadatacheck will have two parts. this is just first part. I'm planning to do both of these checks in the same scrutiny. So there will be only one config variable. So I'm not planning to separate these two scrutinies and have two different config variables.

Reply:
@eolivelli @sijie 

in this bp - https://github.com/apache/bookkeeper/blob/master/site/bps/BP-34-cluster-metadata-checker.md I mentioned that there would be one single checker which would validate placement policy, underreplicated state and availability of entries in bookies. But @jvrao is of the opinion that since PlacementPolicy validation would be light weight it makes sense to run more frequently and have different config intervals for placement policy checker and availability of entries checker. So I'm planning to split this into two different checkers, this one would be placement policy checker and the other (yet to be implemented) one would be for availability of entries (and underreplication state) checker. So I'll rename this metadatachecker to placementpolicychecker and other corresponding method names and config variables.